### PR TITLE
Initial Text4Shell blog post

### PR DESCRIPTION
**Pull request by freeqaz**
Please give this a review so that we can ship this out. Thanks!

### CODE REVIEW

File: docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+Note: While the vulnerability is also known as "Act4Shell", we'll be referring to it as "Text4Shell" in this article.
+
+## Who is impacted?
+
+Applications that use the Apache Commons Text library (or that depend on libraries using it) are impacted.
+
+While the name Text4Shell may suggest that this issue is similar to the
+[Log4Shell vulnerability](https://lunasec.io/docs/blog/log4shell-cve-2021-44228/), that is fortunately not the case.
+
+Text4Shell is fortunately similar to [Spring4Shell](https://www.lunasec.io/docs/blog/spring-rce-vulnerabilities/) in
```
Comment (ajvpot) [inline docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx:44]: too many fortunatelys

**Review by factoidforrest — APPROVED** [review]
Straight and to the point

File: docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+### Proof-of-Concept Code
+
+There are a few excellent resources available for this vulnerability already, including the original
+[GitHub Advisory](https://securitylab.github.com/advisories/GHSL-2022-018_Apache_Commons_Text/) and a
+[Docker POC on GitHub](https://github.com/karthikuj/cve-2022-42889-text4shell-docker), and this post will just be
+consolidating the information in those resources together.
+
+The following code snippet is a proof-of-concept that demonstrates how the vulnerability can be exploited.
+
+```typescript
```
Comment (ajvpot) [inline docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx:70]: this is Java not ts?

File: docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx (hunk 1 above)
Comment (freeqaz) [reply docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx:44]: thx

File: docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx (hunk 2 above)
Comment (freeqaz) [reply docs/blog/2022-10-19-text4shell-cve-2022-42889.mdx:70]: syntax highlighting in docusaurus doesn't work for `java` for some reason. This was my quick hack for Log4Shell lol

**Pull request merged**